resources/language/resource.language.en_gb/strings.po

```diff
@@ -297,3 +300,6 @@ msgctxt "#30907"
 msgid "Remove Widevine CDM library..."
 msgstr ""
 
+msgctxt "#30909"
+msgid "Custom Path"
```

Comment:
Please use `Custom temporary directory` consistently when you interact with the user.